ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/util/ConcurrencyLimiterTest.java

```diff
@@ -31,35 +36,88 @@
 
   @Test
   public void shouldSucceedUnderLimit() {
-    // Given:
```

Comment:
I went ahead and dropped these form fields since we weren't really following the pattern anyway (notice how we do more stuff in the "Then" section and then make more assertions).